common/views/components/SearchFilters/SearchFilters.tsx

```diff
@@ -40,6 +41,11 @@ export type SearchFiltersSharedProps = Props & {
   locationsTypeInUrlArray: string[];
   imagesColor: string | null;
   aggregations?: CatalogueAggregations;
+  enableMoreFilters: boolean;
+  languagesInUrl: string[];
+  subjectsInUrl: string;
+  genresInUrl: string;
```

Comment:
The more I read this - the more we should really just be passing the props around from the `GetServerSideProps` method... We can do this in the refactor.

Reply:
But can we just leave them called `languages` without the `InUrl`?

Reply:
Yeah just looking at this now and the existing functionality ... the way it currently works 

Search Filters -> gets the params from worksRouteProps -> which then trickles down to lower components in sharedFilteredProps. 

I think it would be better for now just to rename the variables as these props are going to used in other components and then we can figure out if we need todo a bigger refactor.

  const languagesInUrl: string[] = worksRouteProps?.languages || [];
  const subjectsInUrl: string = worksRouteProps?.subjectsLabel || '';
  const genresInUrl: string = worksRouteProps?.genresLabel || '';
  const workTypeInUrlArray = worksRouteProps.workType || [];
  const locationsTypeInUrlArray = worksRouteProps.itemsLocationsType || [];